docs/graphx-programming-guide.md

```diff
@@ -726,7 +726,7 @@ class GraphOps[VD, ED] {
     var g = mapVertices( (vid, vdata) => vprog(vid, vdata, initialMsg) ).cache()
 
     // compute the messages
-    var messages = g.mapReduceTriplets(sendMsg, mergeMsg)
+    var messages = GraphXUtils.mapReduceTriplets(g, sendMsg, mergeMsg)
```

Comment:
In the PR description, could you mention [SPARK-12995](https://github.com/apache/spark/pull/10918) for the removal at Spark 2.0, @WeichenXu123 ?